modules/downloading-pipeline-run-step-logs.adoc

```diff
@@ -0,0 +1,47 @@
+:_module-type: PROCEDURE
+
+[id="downloading-pipeline-run-step-logs_{context}"]
+= Downloading pipeline run step logs
+[role='_abstract']
+Instead of viewing the step logs of a pipeline run using the log viewer on the {productname-short} dashboard, you can download them for further analysis. You can choose to download the logs belonging to all steps in your pipeline, or you can download the log only for the step log displayed in the log viewer.
+
+.Prerequisites
+* You have installed the OpenShift Pipelines operator.
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and contains a pipeline server.
+* You have imported a pipeline to an active and available pipeline server.
+* You have previously triggered a pipeline run.
+
+.Procedure
+. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Pipelines*.
++
+The *Pipelines* page opens.
+. From the *Project* list, select the project whose pipeline run step logs you want to view.
+. For the pipeline that you want to see logs for, click *Expand* (image:images/rhoai-expand-icon.png[]).
+. Click *View runs* on the row containing the pipeline version whose run logs you want to view.
```

Comment:
Fixed.

Reply:
Regarding your comment, this should actually be: 

Click _View runs_ on the row containing the pipeline version that you want to download logs for.

The module relates to downloading a log, rather than just viewing it.